src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -312,39 +323,39 @@ object ConductrPlugin extends AutoPlugin {
         features.flatMap(Flags.feature +: _) ++
         ports.withFlag(Flags.port) ++
         logLevel.withFlag(Flags.logLevel) ++
-        conductrRoles.map(_.asConsoleNArg).withFlag(Flags.conductrRole) ++
+        conductrRoles.flatMap(r => if (r.nonEmpty) Flags.conductrRole +: r.toSeq else r) ++
```

Comment:
Roles weren't being passed properly to `conduct-cli` - they would be concatenated as a single string separated by ` ` instead of as separate arguments.

```
Before: Array("--conductr-role", "role1 role2")
After:  Array("--conductr-role", "role1", "role2")
```